src/molecule.h

```diff
@@ -16,6 +16,8 @@
 
 #include "pointers.h"
 
+#include "json.h"
```

Comment:
Please see my general comment on this and the need to use a PIMPL pattern elsewhere, but also the comment *inside* of json.h which says to **not** include this in a header.